backend/src/util.rs

```diff
@@ -180,6 +180,27 @@ pub struct Price {
     pub price_pupil: u32,
 }
 
+/// This struct contains all environmental information. co2 in gramms, water in litres
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct EnvironmentInfo {
```

Comment:
is this the right place? or should it go to the parser model?

Reply:
I mean, we need it for the graphql thing right?

Reply:
For graphql to make this struct quariable we need to create a separate version anyways. Only for enums it is more convenient that way (a hassle to duplicate them)